website/blog/2023-12-11-semantic-layer-on-semantic-layer.md

```diff
@@ -0,0 +1,98 @@
+Once the dust settles and the product is in the wild, the next thing the team needs to do is track how the product is doing. 
+How many users do we have? How is performance looking? What features are customers using? How often? Answering these questions is vital to understanding the success of any product launch.
+
+At dbt we recently made the [Semantic Layer Generally Available](https://www.getdbt.com/blog/new-dbt-cloud-features-announced-at-coalesce-2023). The Semantic Layer lets teams define business metrics centrally, in dbt, and access them in multiple analytics tools through our semantic layer APIs. 
+I’m a Product Manager on the Semantic Layer team, and the launch of the Semantic Layer put our team in an interesting, somewhat “meta,” position: we need to understand how a product launch is doing, and the product we just launched is designed to make defining and consuming metrics much more efficient.  It’s the perfect opportunity to put the semantic layer through its paces for product analytics. This blog post walks through the end-to-end process we used to set up product analytics for the dbt Semantic Layer using the dbt Semantic Layer. 
+
+## Getting your data ready for metrics
+
+The first steps to building a product analytics pipeline with the Semantic Layer look the same as just using dbt - it’s all about data transformation. The steps we followed were broadly:
+
+1. Work with engineering to understand the data sources. In our case, it’s db exports from Semantic Layer Server.
+2. Load the data into our warehouse. We use Fivetran and Snowflake. 
+3. Transform the data into normalized tables with dbt. This step is a classic. dbt’s bread and butter. You probably know the drill by now.
+
+There are [plenty of other great resources](https://docs.getdbt.com/docs/build/projects) on how to accomplish the above steps, I’m going to skip that in this post and focus on how we built business metrics using the Semantic Layer.  Once the data is loaded and modeling is complete, our DAG for the Semantic Layer data looks like the following:
+
+![Screenshot 2023-09-28 at 4.05.25 PM.png](../static/img/blog/2023-12-11-semantic-layer-on-semantic-layer/Screenshot-dag.png)
+
+*Good old-fashioned debt DAG, with an extra node at the end*
+
+
+
+Let’s walk through the DAG from left to right: First, we have raw tables from the Semantic Layer Server loaded into our warehouse, next we have staging models where we apply business logic and finally a clean, normalized `fct_semantic_layer_queries` model. Finally, we built a semantic model named `semantic_layer_queries` on top of our normalized fact model. This is a typical DAG for a dbt project that contains semantic objects. Now let’s zoom in to the section of the DAG that contains our semantic layer objects and look in more detail at how we defined our semantic layer product metrics. 
+
+## How we build semantic models and metrics
```

Comment:
Do we want to link to the "How we build our metrics" best practices doc in this section? Bonus points if we remove the beta tag in that guide

Reply:
Done! Where is the beta tag? I can't find it.

Reply:
he might mean [this `:::tip:::` callout](https://docs.getdbt.com/best-practices/how-we-build-our-metrics/semantic-layer-1-intro)

Reply:
Ah I see it now. Removed!